barretenberg/cpp/src/barretenberg/benchmark/basics_bench/basics.bench.cpp

```diff
@@ -364,6 +364,65 @@ void sequential_copy(State& state)
         }
     }
 }
+
+/**
+ * @brief Evaluate how much uint256_t multiplication costs (in cache)
+ *
+ * @param state
+ */
+void uint_multiplication(State& state)
```

Comment:
Used to understand how much faster 29-limb version is. In wasmer it is twice as fast